src/summaries/smart_contracts.yaml

```diff
@@ -0,0 +1,159 @@
+- title: Working of Ethereum Transactions
+  shortTitle: Ethereum Working
+  key: ethereum-transactions-working
+  details: |
+    - How Ethereum Transaction work?
+      * Transactions involve sending data and/or ETH (or other tokens)
+      from one address to another.
+      * An Ethereum user can control addresses through an externally
+      owned account (EOA) or by using smart contract code (contract
+      account).
+      * When sending data to a contract account, the data are used to
+      execute code in that contract.
+      * Transactions sent to an EOA can only transfer ETH.
```

Comment:
What is `EOA`? Might not be clear to the end user.

Reply:
I have provided the full form and also a basic definition in the above point.